libs/community/langchain_community/chat_models/minimax.py

```diff
@@ -127,7 +127,6 @@ def _default_params(self) -> Dict[str, Any]:
             **self.model_kwargs,
         }
 
-    _client: Any
```

Comment:
why this change

Reply:
Previously I thought that `_client` was not used in this code, but now I think it may be used as a connection object in the future to reduce the number of socket connections (no need to recreate the socket connection every time `invoke`)----If this is necessary, I will submit a PR later to complete this function?

Reply:
`_client` is not used here, so I deleted it.